<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchWidget</class>
 <widget class="QWidget" name="SearchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1016</width>
    <height>181</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="searchGroup">
     <property name="title">
      <string>Search</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="ricercaLayout">
      <item>
       <layout class="QHBoxLayout" name="searchTop">
        <item>
         <widget class="QPushButton" name="closeSearchPanel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="toolTip">
           <string>Close search panel</string>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string>border-color:transparent;
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../risorse.qrc">
            <normaloff>:/quit/images/window-close.png</normaloff>:/quit/images/window-close.png</iconset>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblXPath">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 0, 0);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text">
           <string>XPath</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabel">
          <property name="text">
           <string>Text to &amp;Search</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>searchBox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="searchBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="clearText">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../risorse.qrc">
            <normaloff>:/editText/images/edit-clear-locationbar-rtl.png</normaloff>:/editText/images/edit-clear-locationbar-rtl.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="cmdSearchlets">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../risorse.qrc">
            <normaloff>:/open/images/document-open.png</normaloff>:/open/images/document-open.png</iconset>
          </property>
          <property name="popupMode">
           <enum>QToolButton::MenuButtonPopup</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="info">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ctrl+Space to activate the autocompletion using element and attribute names.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../../risorse.qrc">:/xsdimages/documentinfo</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="findButton">
          <property name="text">
           <string>Find</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="counting">
          <property name="text">
           <string>Count</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="useXQuery">
          <property name="text">
           <string>XPath</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cmdOpenAdvancedPanel">
          <property name="text">
           <string>Open panel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="labelScope">
          <property name="text">
           <string>scope:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="searchScope">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="searchLabel">
          <property name="text">
           <string>Search in:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="searchLocation">
          <property name="editable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="isMatchExactValue">
          <property name="text">
           <string>Match Exact Value</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="isCaseSensitive">
          <property name="text">
           <string>Case Sensitive</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="isOnlyChildren">
          <property name="text">
           <string>Only in Children</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="isHiglightAll">
          <property name="text">
           <string>Find All</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="searchBottom">
        <item>
         <widget class="QLabel" name="searchResult">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>searchResults:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="occNumber">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblSize">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Size:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="sizeOfSearch">
          <property name="text">
           <string>---</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="cmdNext">
          <property name="text">
           <string>Find next</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="wrapAround">
          <property name="text">
           <string>Wrap around</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="closeUnrelated">
          <property name="text">
           <string>Close Unrelated</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="selectionToBookmarks">
          <property name="text">
           <string>Selection to Bookmarks</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="showSize">
          <property name="text">
           <string>Show Size</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../risorse.qrc"/>
  <include location="../../risorse.qrc"/>
 </resources>
 <connections/>
</ui>
